fungal-acne.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>

```

Comment:
Another one of my fully written up articles with complete photos, text, and video.